iRecruit-express-app/memcache.js

```diff
@@ -1,19 +1,21 @@
 const cache = {};
 
-function get(key) {
+export function get(key) {
+  console.log('Getting data from cache for key:', key);
   return cache[key];
   // retrieves data from the cache based on the given key
 }
 
-function set(key, value, expiration = null) {
+export function set(key, value, expiration = null) {
   // stores data in the cache with the specified key
   // If expiration is provided, set a time-to-live (TTL) for the data
   cache[key] = { value, expiration };
+  console.log(cache[key]);
```

Comment:
This expiration is not doing anything, will you be still implementing the TTL functionality?